Xamarin.Forms.Core/IndicatorView.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using Xamarin.Flex;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[ContentProperty(nameof(IndicatorLayout))]
+	[RenderWith(typeof(_IndicatorViewRenderer))]
+	public class IndicatorView : TemplatedView
+	{
+		public static readonly BindableProperty PositionProperty = BindableProperty.Create(nameof(Position), typeof(int), typeof(IndicatorView), default(int), BindingMode.TwoWay, propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorStyles());
+
+		public static readonly BindableProperty CountProperty = BindableProperty.Create(nameof(Count), typeof(int), typeof(IndicatorView), default(int), propertyChanged: (bindable, oldValue, newValue)
+			=> ((IndicatorView)bindable).ResetIndicatorCount((int)oldValue));
+
+		public static readonly BindableProperty MaximumVisibleCountProperty = BindableProperty.Create(nameof(MaximumVisibleCount), typeof(int), typeof(IndicatorView), int.MaxValue, propertyChanged: (bindable, oldValue, newValue)
```

Comment:
MaximumVisibleXXX (XXX being DOTS, or INDICATORS, but not Count), or, to make it shorter `MaximumVisible`